### PR TITLE
fixes issue with sending PWM signals to ESCs in passthrough mode

### PR DESCRIPTION
**Pull request by ndepal**

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rcS [hunk 1]
```diff
@@ -820,6 +820,8 @@ then
 			fmu mode_pwm4
 		fi
 
+		pwm failsafe -c 1234 -p 900
```
Comment (julianoes) [inline ROMFS/px4fmu_common/init.d/rcS]: This is hardcoding the PWM value which should be read from the PWM_DISARMED parameter.

Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/rcS]: Indeed. Can you adjust and retest @ndepal?

**Comment by ndepal** [comment]
It looks like `PWM_DISARMED` isn't set at that point. Where should that be set?
While we're at it, it should probably also be `-c $PWM_OUT`?

**Comment by julianoes** [comment]
I would like to avoid that any parameters are needed on the Pixfalcon/Pixracer side except `SYS_AUTOSTART`. `PWM_DISARMED`/`PWM_MIN`/`PWM_MAX` are already used on the Snapdragon side.

**Comment by ndepal** [comment]
I agree. But then how can we avoid hardcoding there?

**Comment by LorenzMeier** [comment]
Rebased and applied

**Comment by julianoes** [comment]
Why did you apply jt? I thougt we didn't want the hard-coded value in there?

**Comment by LorenzMeier** [comment]
Because the disarmed pulse is irrelevant during operation and we have by far more important things to work on.

**Comment by julianoes** [comment]
Ok, I just thought it wasn't required for @ndepal.

**Pull request closed without merging**